src/Credit/BillException.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Wearesho\Bobra\Portmone\Credit;
+
+use Wearesho\Bobra\Payments;
+
+/**
+ * Class BillException
+ * @package Wearesho\Bobra\Portmone\Credit
+ */
+class BillException extends Payments\Credit\Exception
+{
+    /** @var string */
+    protected $errorCode;
+
+    public function __construct(
+        Payments\Credit\TransferInterface $transfer,
+        string $errorCode,
+        string $message,
+        int $code = 0,
+        \Throwable $previous = null
+    ) {
+        $this->errorCode = $errorCode;
+
+        parent::__construct($transfer, $message, $code, $previous);
+    }
+
+    public function getErrorCode(): string
+    {
+        return $this->errorCode;
```

Comment:
Why not to use native $exception->getCode() ?

Reply:
because code from Portmone service is not in integer type.
examples:
![image](https://user-images.githubusercontent.com/5941637/44462728-41532d80-a61e-11e8-94c4-1692b0f9149f.png)